ftests/explorer-ftests-base/src/main/java/org/nuxeo/functionaltests/explorer/pages/artifacts/ComponentArtifactPage.java

```diff
@@ -91,18 +91,18 @@ public void checkAlternative() {
         checkAliases(null);
     }
 
-    public void checkVersioningComponent() {
-        checkCommon("Component org.nuxeo.ecm.core.api.versioning.VersioningService",
-                "Component org.nuxeo.ecm.core.api.versioning.VersioningService", "In bundle org.nuxeo.ecm.core",
-                "Documentation\n" + "Aliases\n" + "Resolution Order\n" + "Start Order\n" + "Implementation\n"
-                        + "Services\n" + "Extension Points\n" + "XML Source");
-        checkRequirements(null);
-        checkImplementationText("org.nuxeo.ecm.core.versioning.VersioningComponent");
+    public void checkSchedulerComponent() {
+        checkCommon("Component org.nuxeo.ecm.core.scheduler.SchedulerService",
+                "Component org.nuxeo.ecm.core.scheduler.SchedulerService", "In bundle org.nuxeo.ecm.core.event",
+                "Documentation\n" + "Requirements\n" + "Aliases\n" + "Resolution Order\n" + "Start Order\n" + "Implementation\n"
+                        + "Services\n" + "Extension Points\n" + "Contributions\n" + "XML Source");
```

Comment:
I did not add a check on that. There is a method which does it in the `OperationArtifactPage` only so I assumed this was not necessary. Maybe the list of contributions to a component "moves" more frequently than contribs to XP points